src/main/java/org/tron/common/runtime/vm/program/InternalTransaction.java

```diff
@@ -111,16 +119,15 @@ else if(trxType.equals(TrxType.TRX_CONTRACT_CALL_TYPE)){
 
   public InternalTransaction(byte[] parentHash, int deep, int index,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 9 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2fdPPRVdwyhXK5&open=AWak_C2fdPPRVdwyhXK5)

[ISSUE_KEY]: AWak_C2fdPPRVdwyhXK5